Task04.cpp

```diff
@@ -0,0 +1,58 @@
+#include <iostream>
+
+using std::cout;
+using std::cin;
+using std::endl;
+
+const int NAME_SIZE_MAX = 30;
+
+struct Student {
+    char name[NAME_SIZE_MAX + 1];
+    uint64_t fac_num;
+};
+
+void initStudent(const Student& student) {
+    cout << "Name: " << student.name << endl
+         << "Fac. number: " << student.fac_num << endl;

```

Comment:
The name of the function suggest that you will initialise student, instead you are printing it.